normandy/control/views.py

```diff
@@ -0,0 +1,19 @@
+from django.views import generic
+from django.contrib.auth.mixins import LoginRequiredMixin
+
+from normandy.recipes.models import Recipe
+
+
+class IndexView(LoginRequiredMixin, generic.ListView):
+  template_name = 'control/index.html'
+  context_object_name = 'all_recipes_list'
+  login_url = '/control/login/'
```

Comment:
You could subclass `LoginRequiredMixin` and specify this attribute to make a `ControlMixin` class for all control views that redirects to the right URL.